docs/xtrabackup_bin/restoring_individual_tables.md

```diff
@@ -12,59 +12,47 @@ a int(11) DEFAULT NULL
 
 ## Export the table
 
-Created the table in innodb_file_per_table mode, so
-after taking a backup as usual with the –backup option, the
+Created the table in `innodb_file_per_table` mode, so
```

Comment:
To generate an .ibd file in the target directory, create the table using the `innodb_file_per_table` mode.

Reply:
updated